components/expression-editor/components/ExpressionIcon.tsx

```
@@ -1,7 +1,7 @@
 import { cn } from '@/lib/utils';
 import React from 'react';
 import { ColorVariant, IconVariant } from '../utils';
-import { Blocks, Box, Braces, Brackets, Calendar, CircleSlash2, Code, Database, Diamond, FormInput, FunctionSquare, LayoutList, Parentheses, Pyramid, Regex, Signpost, Split, Square, SquareCode, Tag, Terminal, TextCursor, Triangle, Variable } from 'lucide-react';
+import { Ampersands, BadgeCheck, Blocks, Box, Braces, Brackets, Calendar, ChevronLeft, ChevronRight, ChevronsLeft, ChevronsRight, CircleCheckBig, CircleSlash2, ClipboardCheck, Code, CopyCheck, CopyPlus, CopyX, Database, Diamond, Equal, FormInput, FunctionSquare, LayoutList, PackageCheck, Parentheses, Pyramid, Regex, Signpost, Split, Square, SquareCode, Tag, Tags, Tally2, Terminal, TextCursor, Triangle, UserCheck, Variable } from 'lucide-react';
```

ESLint

[error] 'ChevronLeft' is defined but never used.

(@typescript-eslint/no-unused-vars)